src/main/java/com/pnu/dev/radioserviceapi/dto/form/UpdateScheduleItemForm.java

```diff
@@ -0,0 +1,18 @@
+package com.pnu.dev.radioserviceapi.dto.form;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotEmpty;
+
+@Data
+public class UpdateScheduleItemForm {
+
+    private String comment;
+
+    @NotEmpty
+    private String startTime;
+
+    @NotEmpty
+    private String endTime;
```

Comment:
Is it worth to add opportunity to change `dayOfWeek`, as for me, it`s cool for delaying some event to next days

Reply:
Yep, the idea is good.
But I'm not sure users need it right away to start using this app, maybe it's even would be unused, because the time next day should be free to make such update and it also would make UI more complicated.
Let's keep this idea as a future feature possibility for now.
It can be added any time, upon user request.